### ISSUE TITLE
networkInterfaceDefault() takes long time

### ISSUE THREAD

**Issue by esperandote** [sebhildebrandt/systeminformation#411]
**Describe the bug**
networkInterfaceDefault() takes long time (30s) on Macbook Pro 15 sometimes

**To Reproduce**
This problem can be reproduced sometimes, not always.
I actually have found the code that causes the problem:

```
if (_darwin) cmd = 'route get 0.0.0.0 2>/dev/null | grep interface: | awk \'{print $2}\'';
...
let result = execSync(cmd);
```
this command takes long time sometimes, you can just type this command in terminal to check.

**Environment (please complete the following information):**
 - macOS 10.14
 - MacBook Pro 15

**Comment by esperandote** [comment]
This problem also occurs on Macbook Pro 13, macOS 10.15

**Comment by sebhildebrandt** [comment]
@esperandote ... fixed, version 4.27.4 just released
